models/video/video_transcript_events.sql

```diff
@@ -0,0 +1,19 @@
+{{ config(
+    materialized='materialized_view',
+    engine='MergeTree()',
```

Comment:
I think we want this to mirror the others?

```suggestion
    engine=get_engine('ReplacingMergeTree()'),
```

Reply:
Ah yes, thanks for catching that!